### ISSUE TITLE
Add feature to retrieve file upload progress when executing a workflow with run

### ISSUE THREAD

**Issue by ZimmerA** [galaxyproject/planemo#1223]
I often find myself wondering if a file upload is progressing or how much time is left until completion.
It would be great if the run command had a flag that  enables a display of the file upload progress.

**Comment by simonbray** [comment]
If you meant a progress bar for an individual file upload, that is probably not possible, as Galaxy itself doesn't provide progress information for jobs.

If you meant a progress bar for overall progress of file uploads (e.g. 4/10 uploads complete), that could probably be implemented.

**Comment by ZimmerA** [comment]
Yes I was talking about the first approach. Too bad it is not possible. I think the second approach could be useful, although we can already see this on the web interface.
